repl/evaluator_test.go

```diff
@@ -86,7 +97,24 @@ func TestLetError(t *testing.T) {
 		t.Fatalf("NewEvaluator() failed with: %v", err)
 	}
 
-	err = eval.AddLetVar("y", "z + 1")
+	err = eval.AddLetVar("y", "z + 1", nil)
+	if err == nil {
+		t.Errorf("eval.AddLetVar() got non-error, wanted error")
+	}
+
+	_, _, err = eval.Evaluate("y")
+	if err == nil {
+		t.Errorf("eval.Evaluate() got non-error, wanted error")
```

Comment:
You could capture `out` here and display what was returned in the error message.

Reply:
done